NickvisionTagger.GNOME/Blueprints/window.blp

```diff
@@ -356,69 +356,107 @@ Adw.ApplicationWindow _root {
                           };
                         }
 
-                        Adw.PreferencesGroup {
-                          Adw.EntryRow _filenameRow {
-                            title: _("File Name");
-                          }
+                        Gtk.Label _durationFileSizeLabel {
+                          margin-bottom: 6;
+                          label: "00:00:00 • 0 MB";
```

Comment:
I know it's just a placeholder, so can be even empty string, but if you want consistency with real result string, then it should be `MiB`

Reply:
i just left it empty